### DISCUSSION TITLE
[Laravel Bridge] SQS not working.

### DISCUSSION THREAD

**Discussion by mariuszmalek** [brefphp/bref#941] [Support] [answered]
Hello, I'm trying to run SQS but nothing works. I keep getting an error when I try to let Job go. Of course, I created an IAM to which I added full SQS access. I created worker.php and added it to the AWS_SESSION_TOKEN configuration file.

![image](https://user-images.githubusercontent.com/26696860/120760140-0fcc5800-c514-11eb-8ce5-b70cb4b2119a.png)

![image](https://user-images.githubusercontent.com/26696860/120760291-3db19c80-c514-11eb-8195-6bcb85e0487a.png)

![image](https://user-images.githubusercontent.com/26696860/120760829-d9430d00-c514-11eb-934e-6b4cba830aa5.png)

**Comment by deleugpn** [comment, marked as answer]
What is the error message that you're getting? Have you looked into the cloudwatch logs? Can you also share the IAM role that you're using?

**Reply by mariuszmalek** [reply]
Now I noticed in the logs that I have this error: Facade \ Ignition \ Exceptions \ ViewException: Undefined variable: count. I removed "count" variable and now I have that error "Service Unavailable". All logs I have clear, without any errors. Still not working..

**Reply by mariuszmalek** [reply]
https://user-images.githubusercontent.com/26696860/120771994-19f45380-c520-11eb-8083-7823ca96d435.png
https://user-images.githubusercontent.com/26696860/120774860-fa125f00-c522-11eb-80b5-5bda37b6c867.png
